suites/selftest/ts/minimal/multi_line_indent.c

```diff
@@ -0,0 +1,44 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/** @file
+ * @brief Minimal test
+ *
+ * Minial test scenario.
+ *
+ * Copyright (C) 2025 OKTET Ltd. All rights reserved.
```

Comment:
In new files the copyright should be in a separate comment on the second line after the license plate.